.rubocop.yml

```diff
@@ -19,6 +19,7 @@ AllCops:
     - 'tool/**/*'
     - 'truffleruby/target/**/*'
     - 'truffleruby-gem-test-pack-*/**/*'
+    - 'src/main/c/syslog/**'
```

Comment:
Small thing, but I'm trying to keep those in alphabetic order.
Actually, could you ignore `src/main/c/*/extconf.rb`?
There should not be other .rb files under `src/main/c`.